xapian-applications/omega/templates/query

```diff
@@ -1,4 +1,5 @@
-$httpheader{Content-Type,text/html; charset=utf-8}<html>
+$httpheader{Content-Type,text/html; charset=utf-8}
+<html>
```

Comment:
Another point it's useful to make here is what a commit changes ought to match what its commit message says it changed - the message is "Changed all the HTML tags and attributes to lowercase." and a whitespace change doesn't fit that.  Trying to track down the reason a change was made (e.g. "why does the HTML output start with a blank line?") is made much harder if commits mix in unrelated changes.

If the whitespace "fix" had been done as a separate "clean up whitespace" commit, it would be clearer what the author thought they were doing.

Also, commit messages are generally in the present tense - i.e. "Change all [,,,]" is preferred to "Changed all [...]"